pkg/cluster/lcm/health.go

```diff
@@ -79,6 +79,10 @@ type CheckOptions struct {
 	// Name of the storage driver
 	// Applicable for minIO
 	StorageDriver string
+
+	// For s3 options
+	S3Region   string
```

Comment:
Missed functional options? @soulseen

Reply:
`svcConfigGetter` for storage also needs to be implemented, right? or the health check will be blocked.

Reply:
yes, I have added some options. And `svcConfigGetter` for s3 is updated.